crates/sequencer/src/sequencer.rs

```diff
@@ -1012,28 +1009,28 @@ where
         Ok(())
     }
 
-    pub async fn compare_commitments_from_db(
-        &self,
-        pg_connector: PostgresConnector,
-    ) -> Result<(), anyhow::Error> {
-        let ledger_commitment_l1_height =
-            self.ledger_db.get_last_sequencer_commitment_l1_height()?;
-
-        let commitment = pg_connector.get_last_commitment().await?;
-        // check if last commitment in db matches sequencer's last commitment
-        if let Some(db_commitment) = commitment {
-            // this means that the last commitment in the db is not the same as the sequencer's last commitment
-            if db_commitment.l1_end_height as u64
-                > ledger_commitment_l1_height.unwrap_or(SlotNumber(0)).0
-            {
-                self.ledger_db
-                    .set_last_sequencer_commitment_l1_height(SlotNumber(
-                        db_commitment.l1_end_height as u64,
-                    ))?
-            }
-        }
-        Ok(())
-    }
+    // pub async fn compare_commitments_from_db(
+    //     &self,
+    //     pg_connector: PostgresConnector,
+    // ) -> Result<(), anyhow::Error> {
+    //     let ledger_commitment_l1_height =
+    //         self.ledger_db.get_last_sequencer_commitment_l1_height()?;
+    //
+    //     let commitment = pg_connector.get_last_commitment().await?;
+    //     // check if last commitment in db matches sequencer's last commitment
+    //     if let Some(db_commitment) = commitment {
+    //         // this means that the last commitment in the db is not the same as the sequencer's last commitment
+    //         if db_commitment.l2_start_block_number as u64
+    //             > ledger_commitment_l1_height.unwrap_or(SlotNumber(0)).0
+    //         {
+    //             self.ledger_db
+    //                 .set_last_sequencer_commitment_l1_height(SlotNumber(
+    //                     db_commitment.l1_end_height as u64,
+    //                 ))?
+    //         }
+    //     }
+    //     Ok(())
+    // }
```

Comment:
we need to restore this.

on a sequencer crash the sequencer uses this table to restore recent commitments. can explain more in slack if you want.